labrad/oauth.py

```diff
@@ -90,19 +94,20 @@ def get_token(client_id, client_secret, headless=False, timeout=60):
     """
     cached = _get_cached_token(client_id)
     if cached is None:
-        print('No cached token. Logging in...')
+        log.debug('No cached token. Logging in. client_id=%s', client_id)
```

Comment:
Is there a reason for doing it this way vs `s"No cached token. Logging in. client_id=$client_id"`?